include/rmm/device_buffer.hpp

```diff
@@ -421,7 +431,7 @@ class device_buffer {
   {
     _size     = bytes;
     _capacity = bytes;
-    _data     = (bytes > 0) ? memory_resource()->allocate(bytes, stream()) : nullptr;
+    _data     = (bytes > 0) ? _mr.allocate_async(bytes, stream()) : nullptr;
```

Comment:
I just realized, that we could pass our desired alignment in here if we wanted to

Reply:
Seems weird to specify it at the container level rather than the resource level. All device memory resources allocate 256-byte aligned memory, to match what cudaMalloc does.